packages/svelte-query/tests/createInfiniteQuery/createInfiniteQuery.svelte.test.ts

```
@@ -1,11 +1,10 @@
-import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
-import { QueryClient } from '@tanstack/query-core'
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
+import { render } from '@testing-library/svelte'
 import { fireEvent, render } from '@testing-library/svelte'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'render'. Consider to delete it or rename it.

'render' is defined here:


(lint/suspicious/noRedeclare)